app/controllers/application_controller.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
-  before_filter :set_default_locale
-  after_filter :set_response_headers
+  before_action :set_default_locale
+  after_action :set_response_headers
```

Comment:
There's nothing wrong w/ this change, but don't worry about fixing other deprecations that you encounter while testing. We've batched the deprecations into tickets, so that each PR will address a specific type of deprecation to simplify code reviews.